kion/internal/kionclient/helper.go

```diff
@@ -487,3 +488,75 @@ func PrintHCLConfig(config string) {
 	fmt.Println("Generated HCL configuration:")
 	fmt.Println(config)
 }
+
+// ConvertInterfaceSliceToIntSlice converts a slice of interfaces to a slice of integers.
+func ConvertInterfaceSliceToIntSlice(input []interface{}) []int {
+	// Create a slice of integers with the same length as the input slice.
+	output := make([]int, len(input))
+	// Iterate over the input slice, casting each element to an integer.
+	for i, v := range input {
+		output[i] = v.(int)
+	}
+	return output
+}
```

Comment:
FWIW, this helper already existed; it was just centralized in `[resource_project_enforcement.go](https://github.com/kionsoftware/terraform-provider-kion/blob/main/kion/resource_project_enforcement.go#L462)`, and I moved it out of there so that I could use it here as well.  

I'll look into consolidation.

Reply:
`FlattenIntArray` and `ConvertToIntSlice` aren't used anywhere, so I removed them from the codebase.